_generator/types.yaml

```
@@ -578,3 +578,48 @@ types:
   activitystateenum:
     file: _objects.md
     anchor: activity-state
+  voucherresult:
+    file: vouchers.md
+    anchor: voucherresult
+  voucher:
+    file: vouchers.md
+    anchor: voucher
+  vouchertypeenum:
+    file: vouchers.md
+    anchor: voucher-type
+  vouchercode:
+    file: vouchers.md
+    anchor: voucher-code
+  voucherassignment:
+    file: vouchers.md
+    anchor: voucher-assignment
+  voucherfilterparameters:
+    file: vouchers.md
+    anchor: voucherfilterparameters
+  voucherextent:
+    file: vouchers.md
+    anchor: voucher-extent
+  voucherwriteresult:
+    file: vouchers.md
+    anchor: voucherwriteresult
+  multiplevoucheraddparameters:
+    file: vouchers.md
+    anchor: multiplevoucheraddparameters
+  voucheraddparameters:
+    file: vouchers.md
+    anchor: voucher-parameters
+  multiplevoucherupdateparameters:
+    file: vouchers.md
+    anchor: multiplevoucherupdateparameters
+  voucherupdateparameters:
+    file: vouchers.md
+    anchor: vouchers-update-parameters
+  vouchertypeupdatevalue:
+    file: vouchers.md
+    anchor: vouchertypeupdatevalue
+  guidienumerableupdatevalue:
+    file: vouchers.md
+    anchor: guidienumerableupdatevalue
```

issue: I believe this one should go into _objects: https://mews-systems.gitbook.io/connector-api/operations/_objects#array-of-strings-update-value

While we type it as GUID, it's effectively a string, so this should work:
```suggestion
  guidienumerableupdatevalue:
    file: _objects.md
    anchor: array-of-strings-update-value
```

---

fixed in [this commit](https://github.com/MewsSystems/gitbook-connector-api/pull/622/commits/bec8b97c6533e9e76d064ffe48df23d73363f654)